### ISSUE TITLE
A bit of disparity between charges and grenade explosion sounds? + bit of feedback of the preview branch

### ISSUE THREAD

**Issue by ItsRodrigoAl** [SmartlyDressedGames/Unturned-3.x-Community#4253]
## Explosion sound disparity
When a loud explosion sounds, the first though it comes to your mind is "wow someone is getting raided"

When you explode the Demolition Charge and Bounce charge, this loud sound is played, however when you explode a grenade it is a more calm one since is just a grenade. Now, something about this latest preview branch are the sound between all of those explosions:

**Charges**
Demolition Charge: Loud explosion (ok)
Bounce Charge: Loud explosion (ok)
Precision Charge: Calm explosion (weird?)

**Grenades**
Any grenade: Calm explosion (ok)
Bounce Grenade: Loud explosion (weird?)

Since the Precision Charge was recently buffed a lot and now it will way more used, shouldn't this have the loud sound? I mean, is a Charge. Also, the Bounce Grenade is a grenade, but when it explodes sounds just like a Demolition/Bounce charge, shouldn't be have the calm explosion sound instead?

### Bounce Charges may need now an aditional buff or some changes?
Since you can craft both Bounce Charges and Bounce Grenades with just the same amount of materials, also both making you jump the same, but the charge requiring a detonator, should not be the Bounce Charge getting an explosion buff? Why would you craft a charge one when you can just make it grenade and it will explode just 1 second later just the same as the charge, but without the detonator

Another thing to say about the bounce items things, when you craft a Precision Charge it requires to be a Demolition Charge to make it be a Precision one and vice versa. However, when making a Bounce Charge or Bounce Grenade, you don't need these "steps" since you can craft them both anytime. Should not be good to be able to craft one first and then the other one? Here's my proposal:

First the Bounce Grenade, requiring only x1 Raw Explosive + x1 Wire
For the Bounce Charge, it would require a x1 Bounce Grenade + x1 Raw Explosive

With these steps, you can make them have more sense with all the game crafting system of "this requires this to be this and then be this" instead of "you can get this with in any way". Plus, I'm adding the additional Raw Explosive just in case you agree the Bounce Charge should recibe the explosion buff

If the Bounce Charge doesn't get the buff, then personally I think the direct Bounce Charge craft shouldn't exist, and you could craft it only from the grenade to then charge and vice versa

### Impact-Absorbing Boots
From the vanilla game I've never seen any craft requiring another specific clothe to be crafted. I mean, while you can't actually craft any clothe, you can craft the leather ones from materials. Now, with the new pants, shouldn't be better to also be crafted from materials instead from a specific already crafted clothe? Instead of requiring x1 Cargo Pants, it would require x3 Cloth? Just an idea :P

### Metal freeforms being vulnerable and fuel canister/tank/generator item descriptions
Thanks you Nelson. Thats it.

~~### Precision Charge buff is excessive
Personally I think the buff to the Precision Charges was needed, but it is right now **excessive**: I get if you double the damage against buildables making x2 of damage than the Demolition Charge, but **4000**? Even the Brick wall (the strongest buildable on the vanilla game) have 1650 HP, what will you be destroying with a Precision Charge then? o.O~~
> **Molton and Renaxon already explained me the high tier thing so nevemind this one:)**

**Comment by MoltonMontro** [comment]
> Even the Brick wall (the strongest buildable on the vanilla game) have 1650 HP, what will you be destroying with a Precision Charge then? o.O

Just wanted to note that Metal and Brick structures have High armor tier, meaning their effective HP is actually double by default. (2200 for metal, 3300 for brick.)

**Comment by ItsRodrigoAl** [comment]
> Just wanted to note that Metal and Brick structures have High armor tier, meaning their effective HP is actually double by default. (2200 for metal, 3300 for brick.)

Yup Renaxon also mentioned me this on the forum, thanks both for telling me, I swear I dont know how did I forget about that. Already edited on the main post as well

**Comment by SDGNelson** [comment]
Thanks for the report. I'm not sure how I managed to mess up the precision charge damage like that. 😕 

I agree about the sounds - will do.

I don't think the charges vs grenades need to have different stats just because of the detonator requirement, as they can serve different purposes. It's easier to do precise line ups with the charges than it is with the grenade (at least until projectiles rewrite). A teammate could also be responsible for detonating the charges. Charges can also be used as a trap.

I chose the cargo pants for crafting the boots because they're relatively common on all vanilla maps. Ideally the crafting system would support a wider filter like "any pants item" in the future.
